kuka_kr180_support/package.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>kuka_kr180_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the KUKA KR 180 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for KUKA KR 180 manipulators. This currently includes the r3500_ultra_k only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>KR 180 r3500_ultra_k - Default</li>
+    </ul>
+    <p> Joint limits and maximum joint velocities are based on the information found in the online <a href="https://www.kuka.com/-/media/kuka-downloads/imported/8350ff3ca11642998dbdc81dcc2ed44c/0000325893_en.pdf">datasheet</a>. All urdfs are based on the default motion and joint velocity limits, unless noted otherwise. </p>
```

Comment:
Links tend to get stale so it's preferred to refer to the exact name and revision of the data sheet here instead